fendermint/actors/blobs/src/state.rs

```diff
@@ -25,6 +25,9 @@ use log::{debug, warn};
 use num_traits::{ToPrimitive, Zero};
 use recall_ipld::hamt::{BytesKey, MapKey};
 
+type BlobSourcesResult =
+    anyhow::Result<Vec<(Hash, u64, HashSet<(Address, SubscriptionId, PublicKey)>)>, ActorError>;
```

Comment:
at some point we should make these giant types into structs

Reply:
https://github.com/recallnet/ipc/issues/571